src/components/Badge/Badge.css.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { recipe } from "@vanilla-extract/recipes";
+import { vars } from "../../utils/theme.css";
+
+export const BadgeContainer = recipe({
+  base: {
+    alignItems: "center",
+    borderRadius: 20,
+    display: "inline-flex",
+    fontFamily: "Inter, -apple-system, BlinkMacSystemFont, sans-serif",
```

Comment:
can we use some habitat typography to configure the font here?

Reply:
@GiovanniVisentiniCasavo yep, I'm still working on it that why for the moment is hard-coded